### ISSUE TITLE
Optical switch panel is super basic right now

### ISSUE THREAD

**Issue by royerloic** [ClearControl/halcyon#3]
Here is how it looks right now:
![image](https://cloud.githubusercontent.com/assets/1870994/14669741/7b2365d2-06ea-11e6-8470-1d5f9ef4def7.png)

Ideally it would look like this:

![image](https://cloud.githubusercontent.com/assets/1870994/14669748/856d657e-06ea-11e6-863a-cab3ecec42b3.png)

Just several of them...

**Comment by royerloic** [comment]
Great HongKee,

Is there a way to speed up the animation when pressing the switch?
it’s a bit slow - which looks great but is also distracting and slows down the
user…

> On 26 Apr 2016, at 15:29, HongKee Moon notifications@github.com wrote:
> 
> Closed #3 https://github.com/RTlib/Halcyon/issues/3.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub https://github.com/RTlib/Halcyon/issues/3#event-641479953

Dr.  Loïc Alain Royer

Post-Doc - Myers Lab
Max Planck Institute
of Molecular Cell Biology
and Genetics
Pfotenhauerstr. 108
01307 Dresden

Too short? Here's why: http://emailcharter.org/

**Comment by hkmoon** [comment]
It's done with RTlib/RTlib@afc202dee79f78c35a7302c5c2596674f3631bb6

You can make it faster in IconSwitchSkin.java

``` java
        moveToDeselected = new TranslateTransition(Duration.millis(60), thumb);
        moveToSelected = new TranslateTransition(Duration.millis(60), thumb);
```
